willow/config/fedora.yml

```diff
@@ -12,4 +12,4 @@ production:
   user: fedoraAdmin
   password: fedoraAdmin
   url: http://fedora:8080/rest
-  base_path: /willow_production
+  base_path: /willow_development  # TODO: change this back to production once the data-restore problems are fully resolved
```

Comment:
Can you leave a note in the relevant issue or open a new one? It's probably too important to leave to a TODO note.